$OpenBSD: patch-include_sysinc_h,v 1.1 2013/11/27 19:59:45 sthen Exp $

quick hack to fix building following if.h split; should really have an
autoconf check for it but that's harder because AC_CHECK_HEADERS isn't enough
for this file

--- include/sysinc.h.orig	Wed Nov 27 06:18:34 2013
+++ include/sysinc.h	Wed Nov 27 06:19:08 2013
@@ -308,6 +308,7 @@
 
 #ifdef HAVE_NET_IF_H
 #	include <net/if.h>
+#	include <net/if_var.h>
 #endif
 
 #ifdef HAVE_NET_IF_MIB_H
